pyfar/classes/coordinates.py

```diff
@@ -799,6 +818,10 @@ def _set_cyl(self, azimuth, z, rho, convention='top', unit='rad'):
 
         # convert to radians
         if unit == 'deg':
+            warnings.warn((
+                "'deg' parameter will be deprecated in pyfar 0.8.0 in favor "
+                "of the 'pf.deg2rad' and 'pf.rad2deg"),
```

Comment:
I would suggest this
```suggestion
                "of pyfar.deg2rad and pyfar.rad2deg"),
```